skl2onnx/_supported_operators.py

```diff
@@ -75,7 +75,11 @@
     # 0.19
     from sklearn.preprocessing import Imputer as SimpleImputer
 from sklearn.preprocessing import Binarizer
-from sklearn.preprocessing import Imputer
+try:
```

Comment:
Can you add a explicit version checker here instead of try block, which seems more clear to the user and doesn't mix with the other issue?

Reply:
People usually do it this way: https://github.com/scikit-learn-contrib/sklearn-pandas/blob/master/sklearn_pandas/cross_validation.py#L2. Exception are cheap in python.

Reply:
Other comment: dealing with version number means also dealing with weird version number (.dev0, …) and it may introduce a bug if somebody use a weird version. I prefer to stick to exception.

Reply:
It's your design decision, If you think after sklearn 0.21, Imputer will be not supported, the version checking looks better. If you think the converter still works well even without Imputer, try-block looks more robust. This is the straightforward idea when they read the code.

Reply:
Imputer = SimpleImputer, both existed until 0.21 removed Imputer. I like try catch otherwise the comparison will have to deal version such as x.x.dev0, x.x.rc1, x.x.beta… I would have to write a couple of unit test just to make sure the comparison works in all cases.